tiff/tiff_test.go

```diff
@@ -161,8 +168,15 @@ func testSingle(t *testing.T, order binary.ByteOrder, in input, out output, i in
 	if tg.Count != out.count {
 		t.Errorf("(%v) tag %v component count decode: expected %v, got %v", order, i, out.count, tg.Count)
 	}
-	if !bytes.Equal(tg.Val, out.val) {
-		t.Errorf("(%v) tag %v value decode: expected %v, got %v", order, i, out.val, tg.Val)
+	if tg.Type == DTAscii && in.val != "" {
+		strOut := string(out.val)
+		if tg.strVal != strOut {
+			t.Errorf("(%v) tag %v string value decode: expected %v, got %v", order, i, strOut, tg.strVal)
```

Comment:
use %q for failing strings. if it's failing, it's likely gibberish somehow: binary, null bytes, trailing/leading whitespace, etc.

Reply:
done